server.py

```diff
@@ -122,6 +122,7 @@ def list_str(values):
     init_swagger_documentation(app_svc.application)
 
     if args.fresh:
+        logging.info("Fresh startup: removing server data files")
```

Comment:
may want to add a note here on where to find previous files or just that the old session/data was saved